src/docs/supplies/vendor-list.md

```diff
@@ -4,7 +4,7 @@ slug: 'vendor-list'
 title: 'Supplies - Vendor List'
 ---
 
-**NOTE: Vendors are listed alphabetically, not in order of preference**
+### NOTE: Vendors are listed alphabetically, not in order of preference
```

Comment:
Headings should only be used for headings

Reply:
markdownlint disagrees but I have no opinion